changelog_entry.yaml

```diff
@@ -0,0 +1,4 @@
+- bump: minor
```

Comment:
@Aleida24 just found the issue: the changelog entry just needs to be a list item, rather than an object.